oidc-test/src/main.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { AuthProvider } from "react-oidc-context";
+import App from "./App";
+
+// Enter your OAuth OpenId Connect issuer info below! 
+
+const oidcConfig = {
+  authority: "", // OpenId Connect Authority URI - example for google: https://accounts.google.com
+  client_id: "", // ClientId - example for google would be like... asdfasdfasdfasdfasdfasdf.apps.googleusercontent.com
+  disablePKCE: true, // disable PKCE so we don't need a secret
```

Comment:
```suggestion
  disablePKCE: true, // enable PKCE so we don't need a secret
```

You need to enable PKCE in order to not have to provide a secret with `authorization_code`